docker-compose.yaml

```diff
@@ -210,7 +312,6 @@ services:
       - "l1keystore:/home/user/l1keystore"
       - "config:/config"
     depends_on:
-      - geth
```

Comment:
This is safe to do as we moved geth to initialize before all these other containers in the test-node.bash script